offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonViewModel.groovy

```diff
@@ -37,6 +37,7 @@ class CreatePersonViewModel {
     @Bindable Boolean affiliationValid
 
     ObservableList availableAffiliations
+    Map<String,List<Affiliation>> affiliationToOrganisations
```

Comment:
Maybe introduce a new data type `Organi(s/z)ation`. That would make it clearer what you do here. You could then go ahead and also remove the availabaleAffiliations completely and replace it with availableOrganizations `(List<Organization>)`